unit-tests-java8/src/test/java/org/eclipse/collections/test/map/immutable/ordered/ImmutableOrderedMapTest.java

```diff
@@ -208,16 +216,12 @@ public void Map_computeIfPresent()
     }
 
     @Override
+    @Test
     public void Map_replaceAll()
     {
         Map<Integer, String> map = this.newWithKeysValues(1, "1", 2, "2", 3, "3");
-        // Note: replaceAll may call the lambda for at least one entry before attempting setValue
-        assertThrows(UnsupportedOperationException.class, () -> map.replaceAll((k, v) -> {
-            // Lambda may be called for first entry before UnsupportedOperationException is thrown
-            assertNotNull(k);
-            assertNotNull(v);
-            return v + "modified";
-        }));
+
+        assertThrows(UnsupportedOperationException.class, () -> map.replaceAll((k, v) -> v + "modified"));
```

Comment:
This one was fixed in #1823 but the test still wasn't running until I added the annotation